pkg/ddc/thin/create_volume.go

```diff
@@ -61,5 +66,37 @@ func (t *ThinEngine) createFusePersistentVolumeClaim() (err error) {
 		return err
 	}
 
-	return volumehelper.CreatePersistentVolumeClaimForRuntime(t.Client, runtimeInfo, t.Log)
+	err = volumehelper.CreatePersistentVolumeClaimForRuntime(t.Client, runtimeInfo, t.Log)
+	if err != nil {
+		return err
+	}
+
+	return t.wrapMountedPersistentVolumeClaim()
+}
+
+func (t *ThinEngine) wrapMountedPersistentVolumeClaim() (err error) {
+	dataset, err := utils.GetDataset(t.Client, t.name, t.namespace)
+	if err != nil {
+		return err
+	}
+
+	for _, mount := range dataset.Spec.Mounts {
+		if strings.HasPrefix(mount.MountPoint, common.VolumeScheme.String()) {
+			pvcName := strings.TrimPrefix(mount.MountPoint, common.VolumeScheme.String())
+
+			mountedPvc, err := kubeclient.GetPersistentVolumeClaim(t.Client, pvcName, t.namespace)
+			if err != nil {
+				return err
+			}
+
+			labelsToModify := common.LabelsToModify{}
```

Comment:
What will happen if the original pvc is already wrapped by another dataset? Do we need check it before patch the labels?

Reply:
Added check before patching labels